### PR TITLE
Spell built-in instead of builtin

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

**Comment by josevalim** [comment]
You broke the build. :)

File: lib/elixir/test/elixir/kernel/typespec_test.exs [hunk 1]
```diff
@@ -249,16 +249,16 @@ defmodule Kernel.TypespecTest do
     end
   end
 
-  test "@type when overriding Elixir builtin" do
-    assert_raise CompileError, ~r"type struct\(\) is a builtin type; it cannot be redefined", fn ->
+  test "@type when overriding Elixir built-in" do
+    assert_raise CompileError, ~r"type struct\(\) is a built-in type; it cannot be redefined", fn ->
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/typespec_test.exs]: I think this error comes from Erlang, we cannot quite change them.

Comment (josevalim) [inline lib/elixir/test/elixir/kernel/typespec_test.exs]: Hence the build failures.

Comment (eksperimental) [inline lib/elixir/test/elixir/kernel/typespec_test.exs]: fixed and tested locally.

**Comment by eksperimental** [comment]
ouch!
trusted my instinct on not running yet another test, .....
thank you José

**Pull request merged**